django/db/migrations/autodetector.py

```diff
@@ -1057,7 +1057,10 @@ def generate_altered_managers(self):
             old_model_name = self.renamed_models.get((app_label, model_name), model_name)
             old_model_state = self.from_state.models[app_label, old_model_name]
             new_model_state = self.to_state.models[app_label, model_name]
-            if old_model_state.managers != new_model_state.managers:
+            if old_model_state.managers != [
+                (name, instance) for (name, instance) in new_model_state.managers
+                if name != 'objects' and instance.__class__ != models.Manager
```

Comment:
This seems like a messy way to do it, but we can't do it in the `ModelState` method, as that is used elsewhere.

Specifically, returning an empty list causes a DeprecationError related to the new Manager API stuff.